css/dropdown.css

```diff
@@ -0,0 +1,89 @@
+
+.dropdown {
+    min-width: 15px;
+    position: relative;
+    margin: 5px;
+    font-family: "JetBrains Mono", monospace;
+}
+
+.dropdown * {
+    box-sizing: border-box;
+}
+
+.select {
+    background: #7b61ff;
+    color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items:center;
+    border: 5px solid #7b61ff;
+    border-radius: 5px;
+    padding: 5px 10px;
+    transition: 0.3s;
+    font-weight: bold;
+    max-width: 150px;
+    margin: 0 auto;
+}
+
+.select-clicked {
+    border: 2px solid #7b61ff;
+}
+
+.select:hover{
+    background-color:#5f48e1;
+}
+
+.caret {
+    width:0;
+    height:0;
+    border-left: 5px solid transparent;
+    border-right: 5px solid transparent;
+    border-top: 6px solid white;
+    transition: 0.3s
+}
+
+.caret-rotate {
+    transform:rotate(180deg);
+}
+
+.menu {
+    list-style: none;
+    padding: 5px;
+    background:white;
+    border: 2px solid #7b61ff;
+    border-radius: 5px;
+    color: #7b61ff;
+    font-weight: bold;
+    position: absolute;
+    top: 15px;
```

Comment:
remove this line: on my screen it looks not correct

<img width="304" alt="Screenshot 2024-09-21 at 4 42 24 PM" src="https://github.com/user-attachments/assets/ae999b60-166f-4e3e-8861-c689cca237de">